### PR TITLE
Verbose when mounting default bind points

### LINKED ISSUE #1992: Document all directories and files which are automatically mounted
Singularity documentation should clearly lay out exactly which directories/files are automatically bind-mounted into the container when running `singularity run/exec/shell ...`. We should be able to explicitly call out these files when running with any combination of flags **e.g. --contain**

### PR DESCRIPTION
**Pull request by WestleyK**
## Description of the Pull Request (PR):

This PR will add a verbose message when mounting a default bind point.

### This fixes or addresses the following GitHub issues:

- Fixes #1992 

<br>

The default bind points will also be documented in the Userdocs.

### CODE REVIEW

File: internal/pkg/runtime/engines/singularity/container.go [hunk 1]
```diff
@@ -1588,6 +1591,7 @@ func (c *container) addLibsMount(system *mount.System) error {
 		sessionFilePath, _ := c.session.GetPath(sessionFile)
 
 		err := system.Points.AddBind(mount.FilesTag, lib, sessionFilePath, flags)
+		sylog.Verbosef("Default mount: /libs:/libs")
```
Comment (WestleyK) [inline internal/pkg/runtime/engines/singularity/container.go]: Correct me if I'm wrong, it seems like `/lib` is not a default mount point with the default singularity config file...
*If thats the case, I should probably remove this line.*

**Comment by WestleyK** [comment]
Alright! I hope I got all the default bind points (`/lib` in question). If I missed any please let be know :smile:

File: internal/pkg/runtime/engines/singularity/container.go (hunk 1 above)
Comment (cclerget) [reply internal/pkg/runtime/engines/singularity/container.go]: @WestleyK Right, this code is reached when used with `--nv` and the final destination is `/.singularity.d/libs`

File: internal/pkg/runtime/engines/singularity/container.go [hunk 2]
```diff
@@ -903,6 +903,7 @@ func (c *container) addKernelMount(system *mount.System) error {
 		if err != nil {
 			return fmt.Errorf("unable to add proc to mount list: %s", err)
 		}
+		sylog.Verbosef("Default mount: /proc/:/proc/")
```
Comment (cclerget) [inline internal/pkg/runtime/engines/singularity/container.go]: Just cosmetic by removing trailing `/`

File: internal/pkg/runtime/engines/singularity/container.go [hunk 3]
```diff
@@ -923,6 +924,7 @@ func (c *container) addKernelMount(system *mount.System) error {
 		if err != nil {
 			return fmt.Errorf("unable to add sys to mount list: %s", err)
 		}
+		sylog.Verbosef("Default mount: /sys/:/sys/")
```
Comment (cclerget) [inline internal/pkg/runtime/engines/singularity/container.go]: same

File: internal/pkg/runtime/engines/singularity/container.go [hunk 4]
```diff
@@ -1147,6 +1149,7 @@ func (c *container) addDevMount(system *mount.System) error {
 		if err != nil {
 			return fmt.Errorf("unable to add dev to mount list: %s", err)
 		}
+		sylog.Verbosef("Default mount: /dev/:/dev/")
```
Comment (cclerget) [inline internal/pkg/runtime/engines/singularity/container.go]: same

File: internal/pkg/runtime/engines/singularity/container.go [hunk 5]
```diff
@@ -1453,11 +1456,13 @@ func (c *container) addTmpMount(system *mount.System) error {
 
 	if err := system.Points.AddBind(mount.TmpTag, tmpSource, "/tmp", flags); err == nil {
 		system.Points.AddRemount(mount.TmpTag, "/tmp", flags)
+		sylog.Verbosef("Default mount: /tmp/:/tmp/")
```
Comment (cclerget) [inline internal/pkg/runtime/engines/singularity/container.go]: same

File: internal/pkg/runtime/engines/singularity/container.go [hunk 6]
```diff
@@ -1453,11 +1456,13 @@ func (c *container) addTmpMount(system *mount.System) error {
...
 	} else {
 		return fmt.Errorf("could not mount container's /tmp directory: %s %s", err, tmpSource)
 	}
 	if err := system.Points.AddBind(mount.TmpTag, vartmpSource, "/var/tmp", flags); err == nil {
 		system.Points.AddRemount(mount.TmpTag, "/var/tmp", flags)
+		sylog.Verbosef("Default mount: /var/tmp/:/var/tmp/")
```
Comment (cclerget) [inline internal/pkg/runtime/engines/singularity/container.go]: same

**Review by cclerget — CHANGES_REQUESTED** [review]

**Review by cclerget — APPROVED** [review]

**Pull request merged**